test/connectors/httpParsers/VatObligationsHttpParserSpec.scala

```diff
@@ -80,6 +79,114 @@ class VatObligationsHttpParserSpec extends SpecBase {
       }
     }
 
+    "the http response status is 200 OK and matches expected Schema if no data" should {
+
+      val testObligations: VatObligations =
+        VatObligations(
+          Seq.empty[VatObligation]
+          )
+
+
+      val responseJson: JsValue = Json.parse(
+        """{
+          |    "obligations": []
```

Comment:
The `obligationDetails` field inside of `obligations` is required by DES so this technically does not match the schema.

Reply:
This is the test that led to me saying it should return a 404 and correcting alll the data.
I don't really need the test now but thought I'd leave it in?
I can remove it now all the data has been fixed as we shouldn't be trying to parse it.